### PR TITLE
Calculate fix fix

### PR DESCRIPTION
**Pull request by parkkoon**
Add test file for a slot.
Change to new Slot Calculate Logic
Replace all slot number Variable with 'Big' format.

### CODE REVIEW

**Review by c0smic0wl — APPROVED** [review]
:owl:

File: app/javascripts/components/playSlot/reducer.jsx [hunk 1]
```diff
@@ -63,10 +63,10 @@ export function reducer(state = PLAY_SLOT_INITIAL_STATE, action) {
...
           .set('isLoading', false)
           .set('hasError', false)
-          .set('minBet', parseFloat(action.payload.minBet))
-          .set('betSize', parseFloat(action.payload.minBet))
-          .set('maxBet', parseFloat(action.payload.maxBet))
-          .set('betUnit', parseFloat(action.payload.minBet))
+          .set('minBet', action.payload.minBet) // Big Number
+          .set('betSize', action.payload.minBet) // Big Number
+          .set('maxBet', action.payload.maxBet) // Big Number
+          .set('betUnit', action.payload.minBet) // Big Number
```
Comment (fish-sushi) [inline app/javascripts/components/playSlot/reducer.jsx:69]: 👍

File: app/javascripts/helpers/web3Service.jsx [hunk 2]
```diff
@@ -169,7 +170,7 @@ class Web3Service {
   }
 
   makeEthFromWei(weiValue) {
-    return parseFloat(this.web3.fromWei(weiValue, 'ether').valueOf(), 10);
+    return Big(this.web3.fromWei(weiValue, 'ether').valueOf());
```
Comment (fish-sushi) [inline app/javascripts/helpers/web3Service.jsx]: Is there any methods for this in Web3.js?

**Review by fish-sushi — CHANGES_REQUESTED** [review]

File: app/javascripts/helpers/web3Service.jsx (hunk 2 above)
Comment (parkkoon) [reply app/javascripts/helpers/web3Service.jsx]: After I try over a method in Web3.js following your advice, I found it.
https://github.com/ethereum/wiki/wiki/JavaScript-API#web3tobignumber
Thxs. I will replace it with that.

**Pull request merged**